<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2D_AddNew_gui</class>
 <widget class="QDialog" name="Graph2D_AddNew_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Graph2D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mathgraphica.qrc">
    <normaloff>:/images/images/graf2d.png</normaloff>:/images/images/graf2d.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_fx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>f(x,t) =</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_expression">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_polarGraph">
     <property name="text">
      <string>polar graph</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../mathgraphica.qrc">:/images/images/graf_2D.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>+x =</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_xmax"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>-x =</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_xmin"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>dx =</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_dx"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Graph color:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_color2D">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_Ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mathgraphica.qrc"/>
 </resources>
 <connections/>
</ui>
